Plugzy.Service/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Azure.Core;
+using Plugzy.Domain.Entities;
+using Plugzy.Infrastructure.Interface.Repository;
+using Plugzy.Models.Base;
+using Plugzy.Models.Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugzy.Service.Services
+{
+    public class AuthenticationService : IAuthenticationService
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly IOtpRepository _otpRepository;
+        public AuthenticationService(IUserRepository userRepository,IOtpRepository otpRepository)
+        {
+            _userRepository = userRepository;
+            _otpRepository = otpRepository;
+        }
+
+        public async Task<CommandResult<Otp>> CheckOtpCode(string phoneNumber)
+        {
+            var checkCode = await _otpRepository.GetAllAsync(c => c.Phone == phoneNumber);
+            var getLastCode = checkCode.OrderByDescending(i => i.ValidTill).FirstOrDefault();
+
+            if (getLastCode != null && getLastCode.ValidTill > DateTime.Now)
+            {
+                getLastCode.LoginTime= DateTime.Now;
+                getLastCode.Attmps = true;
+
+                await _otpRepository.UpdateAsync(getLastCode);
+                return CommandResult<Otp>.GetSucceed(getLastCode);
+            }
+
+            return CommandResult<Otp>.GetFailed("Girdiğin Otp kodu yanlış veya süresi dolmuş lütfen tekrar deneyin.");
```

Comment:
tüm bu ve benzer string response lar için utitlities altında bir "static MessageConstants"   gibi bir sınıf ekleyip  onun içinde yönetir misin,  değişiklik gerektiğinde kolaylık olur  ve aynı ifadeleri kullanma şansın olur